content/en/apps/guides/hosting/vertical-vs-horizontal.md

```diff
@@ -0,0 +1,85 @@
+
+## Vertical scaling in 3.x and 4.x
+
+Before getting into how the CHT horizontally scales, it should be well understood the importance of vertical scaling and what it is.  This is the ability of the CHT to support more users by adding more RAM and CPU to either the bare-metal or virtual machine host. This ensures key services like API, Sentinel and, most importantly, CouchDB, can operate without performance degradation. 
+
+When thousands of users are simultaneous trying to synchronize with the CHT, the concurrency can sometimes overwhelm CouchDB. As discovered [through extensive research](https://forum.communityhealthtoolkit.org/t/how-we-tested-scalability-of-cht-infrastructure/1532) and [large production deployments](https://github.com/medic/cht-core/issues/8324#issuecomment-1691411542), administrators will start to see errors in their logs and end users will complain of slow sync times.  Before moving to more CouchDB nodes, administrators should consider adding more RAM and CPU to the single server where the CHT is hosted. This applies to both CHT 3.x and CHT 4.x. Given the easy of allocating more resources, presumably in virtualized environment like [EC2](https://aws.amazon.com/ec2/), [Proxmox](https://www.vmware.com/content/vmware/vmware-published-sites/us/products/esxi-and-esx.html.html) or [ESXi](https://www.vmware.com/content/vmware/vmware-published-sites/us/products/esxi-and-esx.html.html), this is much easier than moving to [from a single to multi-node CouchDB instance]({{< relref "apps/guides/hosting/4.x/data-migration" >}}).
+
+Here we see a normal deployment following our [bare minimum]({{< relref "apps/guides/hosting/requirements" >}}) hosting requirements for hosting the CHT.  We'll call this a "short" deployment because it is not yet vertically scaled:
+
+```mermaid
+flowchart TD
+
+subgraph couch1[" CouchDB - Single ''short'' Node "]
+    couchInner1["2 CPU/4 GB RAM"]
+end
+
+API["API"] --> HAProxy -->  couch1
+```
+
+After looking at the logs, and seeing error messages about API timeouts to CouchDB, the CHT admin can make this "taller" by adding both more RAM and CPU so it looks like this:
+
+```mermaid
+flowchart TD
+
+subgraph couch2[" CouchDB - Single ''tall'' Node "]
+    couchInner2["16 CPU/16 GB RAM"]
+end
+
+API["API"] --> HAProxy -->  couch2
+```
+
+Since both CHT 3.x and 4.x support this, vertical scaling is an easy, good first step in addressing performance issues in the CHT.
+
+## Horizontal scaling 4.x
+
+When:
+
+* you can no longer vertically scale your CHT instance because of hardware limitations
+* vertically scaling stops yielding better performance
```

Comment:
internally i believe we don't scale beyond `r4.8xlarge` instance sizes. this could be useful info for others

Reply:
ah cools!  That's 32 cores and 244 ram, so put that as a generic stat per 8b6a889  - that work?